docker/terraform.sh

```diff
@@ -17,6 +17,8 @@ BACKEND_NAME="sa${ENVIRONMENT}${RG_LOCATION_SHORT}${SUFFIX}"
 CONTAINER_NAME="tfstate-${DIR}"
 ENVIRONMENT_FILE="/tmp/${ENVIRONMENT}.env"
 
+export HELM_CACHE_HOME=/tmp/.cache 
```

Comment:
Should we use `/tmp/${DIR}/.terraform/.helmcache` ?

Reply:
FIxed.